core/src/main/java/com/amplitude/core/Amplitude.kt

```diff
@@ -130,7 +130,7 @@ open class Amplitude internal constructor(
      * @return the Amplitude instance
      */
     @JvmOverloads
-    fun identify(userProperties: Map<String, Any>, options: EventOptions? = null): Amplitude {
+    fun identify(userProperties: Map<String, Any?>?, options: EventOptions? = null): Amplitude {
```

Comment:
This is for enable ampli identify call to call core identify. 
<img width="979" alt="Screen Shot 2022-06-03 at 12 30 28 PM" src="https://user-images.githubusercontent.com/13028329/171936796-4aa0f4f4-b6cf-4cb1-9a13-2ea4eb56d711.png">
The properties type is Map<String, Any?>? but here required Map<String, Any>. So updated this and add more logic to handle the null.